serverlessworkflow/modules/ROOT/pages/release_notes.adoc

```diff
@@ -0,0 +1,21 @@
+= What's new on {page-component-display-version}
```

Comment:
Are you sure about that ? This will be overriden for each release ?
and so in `main`, we see 
```

Reply:
I mean don't we want to keep history of each release in the release notes ?
I would be easier to people to search in one single release notes page

Reply:
we would just preprend for each release to this page. what do you think ?
Something like 
```
= What's new in Serverless Workflow

== 1.26.0.Final
[...]

== 1.25.0.Final
[...]

...
```

Reply:
Yes, we will change for every release. Users can see in each branch what changed (navigating at the top of the page), rather than a huge asciidoc macaroni page.

Reply:
suppose that I am a user and update to a newer version (let's say from 1.13 to 1.25) and something is broken ...
How to I look for that particular change in the release notes to know when it was changed ?
Do I have to go through all release notes pages ???

Reply:
Yes, you will. Ideally, we should have a Releases page in our GH repos. You can have a whole view of the releases, but not in the docs. 30 releases from now, the page will be huge, not ideal without paging. Docs are supposed to be short.

Reply:
See: https://github.com/quarkusio/quarkus/releases/tag/2.11.2.Final

And https://github.com/quarkusio/quarkus/releases.

Reply:
fair enough

Reply:
We can add the links to previous release notes in this page, wdyt?